src/js/views/admin/program/flow/program-flow.scss

```diff
@@ -98,6 +98,95 @@
   border-top: $black-90 1px solid;
 }
 
+.program-flow__action-name {
+  display: table-cell;
+  vertical-align: middle;
+  white-space: normal;
+}
+
+.program-flow__action-icon {
+  display: table-cell;
+  position: relative;
+  vertical-align: middle;
+
+  .icon {
+    color: color(blue);
+    font-size: 20px;
+    margin-right: 16px;
+    vertical-align: middle;
+    width: 20px;
+  }
+}
+
+.program-flow__action-attachment {
+  border: 1px solid transparent;
+  display: inline-block;
+  padding: 0 8px;
+  vertical-align: middle;
+}
+
+.program-flow__action-ts {
+  float: right;
+  font-size: 12px;
+  margin: 8px 0 0 8px;
+  min-width: 40px;
+  vertical-align: middle;
+}
+
+.program-flow__action-meta {
+  overflow: visible;
+}
+
+.program-flow__action-sort {
+  bottom: -16px;
+  cursor: move;
+  display: none;
+  position: absolute;
+  right: -16px;
+  text-align: center;
+  top: -16px;
+  width: 60px;
+
+  .icon {
+    color: $black-60;
+    font-size: 20px;
+    margin-top: 50%;
+    transform: translateY(-50%);
+    vertical-align: middle;
+    width: 20px;
+  }
+}
+
+.table-list__item:hover {
+  .program-flow__action-sort {
+    display: inline-block;
+  }
+
+  .program-flow__action-ts {
+    display: none;
+  }
+}
+
+.program-flow__list.is-dragging {
```

Comment:
I'm not thrilled about this, there's a lot of overriding going on. But SortableJS doesn't play nice with list items that have hover states. The hover styles will remain applied to whatever item fills the space where the dragged item used to be, creating a confusing UI that makes it difficult to discern where the dragged item will be dropped.

The solution was to add a class to the entire list when an item is being dragged and override the hover styles.

Reply:
`table-list__` classes shouldn't show up in this file at all.  Likely the only thing that should be in this file is some sort of `program-flow__list-item` which would handle hiding the `program-action_ts` on hover.
Then maybe all of the sort stuff can be made generic and we can move it to the `table-list` styles.  Hopefully any other table list _would_ do this the same way, so it seems like it would make sense to put it there.